mypy/checker.py

```diff
@@ -1894,6 +1892,19 @@ def check_init_subclass(self, defn: ClassDef) -> None:
             # all other bases have already been checked.
             break
 
+    def check_enum(self, defn: ClassDef) -> None:
+        assert defn.info.is_enum
+        if defn.info.fullname not in ENUM_BASES:
+            for sym in defn.info.names.values():
+                if (isinstance(sym.node, Var) and sym.node.has_explicit_value and
+                        sym.node.name == '__members__'):
+                    self.fail(
+                        'Cannot declare read-only attribute "__members__"', sym.node
```

Comment:
Can you please move this string to `message_registry.py`?

Reply:
Done, had a bit of a hard time coming up with a reasonable name for the variable. Let me know if you got any better than what I could come up with